### DISCUSSION TITLE
Quarto Extra

### DISCUSSION THREAD

**Discussion by weberam2** [R-nvim/R.nvim#243] [Ideas]
Hello

I'm currently using the R lang extra package to write my [Quarto](https://quarto.org/) (.qmd) files.

However, although it has support for Quarto, it still only focuses on R as the language (Quarto can also process [Python](https://quarto.org/docs/computations/python.html) and [Julia](https://quarto.org/docs/computations/julia.html)).

Would anyone be interested in creating a **Quarto specific Extra package**?

There is a [kickstart setup](https://github.com/jmbuhr/quarto-nvim-kickstarter) for it already, we would just need to adjust the keymappings I think...

I'm still pretty ignorant with neovim and stuff, otherwise I would do it myself (maybe I'll learn...)

Cheers,

Alex W.

**Comment by jalvesaq** [comment]
Until now, I have not needed to deal with big data and use Python or Julia in data analysis. However, other people might be motivated to help you develop the extra functionality you are missing.

**Comment by PMassicotte** [comment]
You can also install quarto.nvim, they are not mutually exclusive.
